### ISSUE TITLE
Addon doesn't show well

### ISSUE THREAD

**Issue by jlramosr** [react-theming/storybook-addon-material-ui#97]
Hello!
my addon is showing of a strange way in my storybook v5.0.11 (the icons and functionality doesn't work well):

![Captura de pantalla de 2019-05-07 12-16-03](https://user-images.githubusercontent.com/6153618/57292722-e5083f00-70c2-11e9-8eb1-ec02e23fc537.png)

What I'm doing bad?

I have another problem. When I create the MUI Theme, I use "override" to add a font with @font-face, but this font doesn't appear in my storybook component. What I have to do about that?

**Comment by UsulPro** [comment]
Hi @jlramosr 

After we switched to the new alpha the UI of this addon wasn't finished yet, so for some sizes it could look weird. But should switch colors well though. It will be likely reworked in the next version. 
If some functionality doesn't work as expected could you let me know details about that? 

When you adding another font are you sure it properly download by components?

**Comment by jlramosr** [comment]
Hi @UsulPro and thanks for the response.

Ok, I see the UI of the same way for all the window sizes and doing click on "clone Theme", "Clear Theme", etc, it doesn't work. I was using v3.9.3 of Material-UI, but with the new version (v.4-beta), this still happens. When I change the theme (I have two themes - dark and light - with different primary colors), the background color of my components it doesn't change. When I use my components outside, this behaviour is correct. I use v5.0.11 of storybook and I've tried with all your v.0.9.0-alpha versions.

![Peek 08-05-2019 08-21](https://user-images.githubusercontent.com/6153618/57353936-5ea73800-716a-11e9-9f2c-1d540150b5fc.gif)

With regard to the fonts, the font downloaded on both themes with:
```
overrides: {
    MuiCssBaseline: {
      '@global': {
        '@font-face': MY_DOWNLOAD_FONT_ARRAY,
      },
    },
},
typography: {
  fontFamily: [ MY_DOWNLOADED_FONT, ... ]
}
```
it doesn't appear in the storybook, but outside the behaviour is correct and the font shows good.

Thanks again!

**Comment by UsulPro** [comment]
@jlramosr sorry for a long answer,
`alpha-19` shows additional errors while theming creation, did you see any warnings in console?

**Comment by stoffeastrom** [comment]
@UsulPro 
I just cloned this https://github.com/indentlabs/notebook-ui and tested and I think it's the same problem? The console:
![image](https://user-images.githubusercontent.com/682243/58432189-73178a00-80b1-11e9-909a-71eed75c7630.png)

**Comment by UsulPro** [comment]
Yes - same!

**Comment by stoffeastrom** [comment]
```
const theme = createMuiTheme({
  palette: {
    primary: purple,
    secondary: green
  },
  status: {
    danger: 'orange'
  }
});

const theme1 = createMuiTheme({
  palette: {
    primary: yellow,
    secondary: blue
  },
  status: {
    danger: 'orange'
  }
});

storiesOf('Button', module)
  .addDecorator(muiTheme([theme, theme1]))
```
and I still get that error when switching themes. Also changing colors in the color picker doesn't update the component

**Comment by stoffeastrom** [comment]
If I do

```js
storiesOf('Button', module)
  .addDecorator(muiTheme([dark, light]))
  .add('with text', a => (
    <Button variant="contained" color="primary">
        test button
    </Button>
```
e.g add `variant="contained"` it starts working

**Comment by UsulPro** [comment]
Great! Addon creates themes from objects you passing to `muiTheme`. So looks like the error comes when we trying to do `createMuiTheme(theme)` twice.

**Comment by UsulPro** [comment]
I suppose I can close this

**Comment by rdurel423** [comment]
@UsulPro I don't think this issue should be closed as the initial post (addon UI not matching what's presented in the README) is still active.

**Comment by goldenshun** [comment]
I also wasn't able to edit the theme and didn't really understand how to use the UI. The UI displayed in the README is what I was hoping for.

**Comment by frankieali** [comment]
I'm attempting to use this addon in my new project. I have the same issue as the original poster in that the latest code does not result in a UI as seen on the project landing page. I was really hoping for the `Show Theme Editor` toggle seen in the demo [here](https://sm-react.github.io/storybook-boilerplate/?theme-ind=1&theme-sidebar=true&theme-full=false&knob-Title=Welcome%20to%20React-Theming&knob-Subtitle=Storybook%20Boilerplate%20Project&knob-Label1=Hello%20Button&knob-Label2=Hello%20Button&selectedKind=Material-UI&selectedStory=Components&full=0&down=1&left=1&panelRight=0&downPanel=sm%2Fstorybook-addon-material-ui%2Fmaterial-panel).

Anyway, I was looking into why the latest version of this project was not working for me and it was because `@storybook/addon-backgrounds` has been updated. Instead of being initialized with `addDecorator` it's now initialized with `addParameters`.

**Comment by frankieali** [comment]
The demo in the README (seen on the project landing page) is version [`0.8.11`](https://github.com/react-theming/storybook-addon-material-ui/tree/0.8.11).

**Comment by UsulPro** [comment]
Yep, after Material-UI released v1 the structure of theme was changed. So I'm considering a new way of how to edit themes. At this moment it's possible to edit theme as a json object from addon panel

**Comment by kaiwa** [comment]
I'm trying to set up alpha 20 with material-ui 3.9.3 and for me as well the component does not update when switching the theme.

I have two themes added in storybook's config.js:

```
addDecorator(muiTheme([
  ATheme
  BTheme,
]));
```

One of the themes is initially successfully applied to the component.

Both themes show up in the addon's theme selector and I can switch them.

When switching, the theme editor shows the selected theme correctly and I see a message is showing in the console with a changing `themeInd`:

```
message arrived at preview sm/storybook-addon-material-ui/material-event/back {themes: Array(2), themeInd: 1}
```

There are no errors besides the typography warning:

```
Warning: Material-UI: you are using the deprecated typography variants that will be removed in the next major release.
Please read the migration guide under https://material-ui.com/style/typography#migration-to-typography-v2
```

But switching the theme has no effect on the component (border color should change to theme.palette.primary.light in my example).

![output](https://user-images.githubusercontent.com/319268/61207540-a65fb800-a6f5-11e9-9859-3ddd82963f97.gif)
